crates/kameo/src/actor/actor_ref.rs

```diff
@@ -140,10 +147,20 @@ impl<A> ActorRef<A> {
     }
 
     /// Sends a message to the actor, waiting for a reply.
-    pub async fn send<M>(
+    ///
+    /// The message reply can be awaited asyncronously, or in a blocking context either by awaiting it directly,
+    /// or calling [`SendResult::blocking_recv()`].
+    ///
+    /// # Example
+    ///
+    /// ```
```

Comment:
You might want to mark this as `no_build` and `no_run`. I don't think this builds or runs on its own so it wont pass `cargo test`.

Reply:
Yeah I might have to do that in another PR since there's a bunch of other code docs failing too when I tried to test just now. So I might deligate that for later